src/xbot/common/math/PID.java

```diff
@@ -18,9 +18,12 @@
     private double m_derivativeValue;
     private boolean errorIsSmall = false;
     private boolean derivativeIsSmall = false;
-    double errorTolerance = -1;
-    double derivativeTolerance = -1;
-
+    double errorTolerance = 0;
```

Comment:
I want to control enable/disable via the SmartDashboard. Since that's already a boolean, I just pass that information down into the PID class.